### PR TITLE
Update: cli update

### PR DESCRIPTION
**Pull request by HuseinA**

### CODE REVIEW

File: obs/libs/utils.py [hunk 1]
```diff
@@ -63,5 +64,31 @@ def human_date(unixtime):
...
+
+
 def compatibility():
+    """get neo compatibility environment"""
     return strtobool(os.environ.get("OBS_USE_NEO"))
+
+
+def user_sanitize(args):
+    """Sanitize input when creating new user"""
+    arg = {}
```
Comment (efenfauzi) [inline obs/libs/utils.py:83]: I thought `args`  has many  args parameter name, this function only for args with key `userId` .  You can define some general function name and pass based on args.

File: obs/cli/storage/bucket.py [hunk 2]
```diff
@@ -91,7 +101,10 @@ def download_object(resource, bucket_name, object_name):
 def upload_object(**kwargs):
     filename = kwargs.get("local_path")
     try:
-        bucket_lib.upload_object(**kwargs)
+        regex = r"[\"\{}^%`\]\[~<>|#]|[^\x00-\x7F]"
```
[the remark this answers is not in the document]
Comment (HuseinA) [reply obs/cli/storage/bucket.py]: done, change ot to "uID"

File: obs/libs/utils.py (hunk 1 above)
Comment (HuseinA) [reply obs/libs/utils.py]: done, change it to "uID"

File: obs/cli/storage/bucket.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (HuseinA) [reply obs/cli/storage/bucket.py]: Done

**Comment by efenfauzi** [comment]
LGTM @anak10thn

**Pull request merged**